feg/gateway/services/s6a_proxy/servicers/ul.go

```diff
@@ -55,6 +64,27 @@ func (s *s6aProxy) sendULR(sid string, req *protos.UpdateLocationRequest, retryC
 	return err
 }
 
+// createULR_Flags creates ULR Flags based on TS 29.272
+func createULR_Flags(req *protos.UpdateLocationRequest) int {
+	// ULR_FLAGS contains defaults flags for ULR-Flags
+	ulrFlags := ULR_FLAGS
+	if req.DualRegistration_5GIndicator {
+		// add TS 29.272 Dual-Registration5G-Indicator on bit 8
+		ulrFlags = ulrFlags | 1<<8
+	}
+	return ulrFlags
+}
+
+// createFeatureListID2 creates the Feature-List-ID 2 based on TS 29.272
+func createFeatureListID2(id2 *protos.FeatureListId2) int {
+	res := 0
+	if id2.NrAsSecondaryRat {
+		// set bit 27 (TS 29.272 Nr As Secondary Rat)
```

Comment:
You could use a bitvector type to clean these things up if you are up for it.

Reply:
I was not able to find a bitvector on the golang standard modules. Is there any I am missing?

I also didn't want to hardcode any structure because the length of bits may change in the future. And also because s6a_proxy has other booleans to signal bits. 

I think that even this requires more code, it is easier to add more flags and see the specific bit positions they are added.

Reply:
Gotcha! sounds good then